hbase_sql/hbase1.2.0/src/main/java/com/splicemachine/hbase/BackupEndpointObserver.java

```diff
@@ -154,7 +156,10 @@ public void prepareBackup(
                 canceled = BackupUtils.backupCanceled();
                 if (!canceled) {
                     // Create a ZNode to indicate that the region is being copied
-                    boolean created = ZkUtils.recursiveSafeCreate(regionBackupPath, HConfiguration.BACKUP_IN_PROGRESS, ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
+                    HRegionInfo regionInfo = region.getRegionInfo();
```

Comment:
We need to apply this change to other platforms too, right?

Reply:
@dgomezferro good catch!